dashboard/test/models/blockly_test.rb

```diff
@@ -143,6 +143,23 @@ def assert_equal_xml(a, b)
     assert_equal xml, xml2
   end
 
+  def create_test_data(name, pool, block_text, args, category: 'Events')
+    [{
+      name: name,
+      pool: pool,
+      category: category,
+      config:
+      {
+        "color" => [140, 1, 0.74],
+        "func" => "atSelectColor",
```

Comment:
why this default?

Reply:
Good catch!  Thanks.  Modified the object to only include key/values that will be used.